pmm-app/src/pmm-update/hooks/usePerformUpdate.test.tsx

```diff
@@ -143,4 +143,57 @@ describe('usePerformUpdate', () => {
 
     wrapper?.unmount();
   });
+
+  it('should increase logOffset value only with values received from server', async () => {
+    let wrapper: ReturnType<typeof mount> | undefined;
```

Comment:
nit: `wrapper` types can be much better if you look at an example in `CheckPanel.test.tsx`. Then we don't to write property guards everywhere.